jupyterlab_hub/typings/xterm/xterm.d.ts

```diff
@@ -0,0 +1,83 @@
+// Type definitions for term.js 0.0.7
```

Comment:
Something seems wrong if we have to copy these typing files into this repo. Comments, @jasongrout?